src/tools/compiletest/src/runtest.rs

```diff
@@ -94,6 +94,8 @@ fn get_lib_name(lib: &str, aux_type: AuxType) -> Option<String> {
             format!("{}.dll", lib)
         } else if cfg!(target_vendor = "apple") {
             format!("lib{}.dylib", lib)
+        } else if cfg!(target_os = "aix") {
+            format!("lib{}.a", lib)
```

Comment:
Remark: this really should use `std::env::consts::{DLL_PREFIX,DLL_EXTENSION}` so it automatically accounts for e.g. `target_os = "aix"`. I can do this in a follow-up PR unless you want to do it in a follow-up.

Reply:
Cool, since I'm new to this part, you are much more expertised than me to handle it generally.

Reply:
No worries, I just realized this because another more experienced contributor pointed out to me std has those constants 😆 ^^

Reply:
Opened https://github.com/rust-lang/rust/pull/127962, blocked on this PR getting merged first.